health/arg_codes.go

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+package health
+
+// ArgCode is a type for the key to be used in the Args of a Warnable.
```

Comment:
Maybe it should be ArgKey if it's documented as a map key?

Reply:
just Arg now